catalog/ui/catalog-ui-search/src/main/webapp/component/sort-item/sort-item.less

```diff
@@ -8,9 +8,15 @@
     margin-top: 10px;
   }
 
-  @{customElementNamespace}sort-item:first-of-type {
+  @{customElementNamespace}sort-item:not(:first-of-type) {
     .sort-remove {
-      display: none;
+      display: inline-block;
+    }
+  }
+
+  @{customElementNamespace}sort-item {
+    .sort-direction {
+      width: ~'calc(50% - @{minimumButtonSize} - 40px)';
```

Comment:
It would not be as simple to understand that its a two part input if we separated them onto separate lines. Any specific concerns you have regarding putting it on a single line?